base_tier_validation/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Base Tier Validation",
     "summary": "Implement a validation process based on tiers.",
-    "version": "11.0.1.1.0",
+    "version": "11.0.1.2.0",
```

Comment:
11.0.2.0.0 instead, remember that you are changing dependencies and are adding migration scripts